src/main/java/seedu/jarvis/ui/MainWindow.java

```diff
@@ -357,11 +357,10 @@ public MissionListPanel getMissionListPanel() {
     private String getLoginMessage() {
         StringBuilder sb = new StringBuilder();
 
-        if (!logic.hasUsername()) {
-            sb.append("Please edit your username using `edit -l u/[USERNAME]` and restart to fetch missions/quests");
+        if (!logic.hasUsername() || !logic.hasPassword()) {
+            sb.append("Please edit your username using `edit -l u/[LUMINUS_USERNAME] [/[PASSWORD]` "
```

Comment:
i think password is missing a /p

Reply:
Perhaps this has to write "Please edit your username and password“?